src/omero/plugins/duplicate.py

```diff
@@ -61,6 +78,44 @@ def cmd_type(self):
         import omero.all
         return omero.cmd.Duplicate
 
+    def _pre_objects(self, parser):
+        parser.add_argument(
+            "--duplicate",
```

Comment:
You decided against the short-cuts?

Reply:
Yes, given that we shortened from `--duplicate-classes` and I imagined you might have initially thought `--duplicate` would be used more often than I'd guess, given that it overrides overrides to put them back to default. We haven't much done it elsewhere, e.g., for `--include` or `--dry-run` or whatever. Could still do it if it brings enough value, maybe capital letters to reduce conflicts with non-graph-command options.